compiler/noirc_evaluator/src/acir/acir_variable.rs

```diff
@@ -1568,6 +1566,33 @@ impl<F: AcirField, B: BlackBoxFunctionSolver<F>> AcirContext<F, B> {
     }
 }
 
+pub(super) fn power_of_two<F: AcirField>(power: u32) -> F {
+    let full_bytes = power / 8;
+    let extra_bits = power % 8;
+    let final_byte: u8 = match extra_bits % 8 {
```

Comment:
It would be helpful to leave a comment that this "final byte" is going to be the first byte in the big endian, or the most significant bit. I stared at it for a while trying to understand why a remainder of 0 becomes 0x01.